energyplus/Model.cc

```diff
@@ -0,0 +1,11 @@
+#include "Model.hh"
+#include "EnergyPlus.hh"
+#include <iostream>
+
+void MyPlantModel::simulate(const PlantLocation &pl) {
+  m.stepForward(heatGain, timeStep);
+}
+
+void MyPlantModel::printResult() const {
+  std::cout << "Model temperature: " << m.temperature << std::endl;
+}
```

Comment:
This file represents the E+ wrapper implementation.  Ideally everything in E+ would magically be 'mocked' in this project, but I don't think that's feasible.  But I think by mocking, say, 5% of the stuff in E+ in our funny EnergyPlus.hh file, we can get a portable model 80% of the way there for implementation in E+ proper.